### PR TITLE
Lizz   dev rhythm 1

### PR DESCRIPTION
**Pull request by lizz7000**

### CODE REVIEW

**Review by billkable — APPROVED** [review]

**Review by billkable — CHANGES_REQUESTED** [review]
The foundations *code patterns* section is incorrect.

See the definition updates inline

File: content/outcomes/application-development-rhythm/foundations.md [hunk 1]
```diff
@@ -145,64 +136,156 @@ Some of the practices you will learn about in this series are:
...
+Some of the methods and techniques you will see in this series includes:
 
 - [Refactoring](https://refactoring.com)
 - Red/Green/Refactor
 
-## Patterns
+## Code Patterns
 
-Patterns are known solutions in the appropriate problem context.
+Code patterns are traceable events that happen when a developer codes.
```
[the remark this answers is not in the document]
Comment (lizz7000) [reply content/outcomes/application-development-rhythm/foundations.md:152]: Thank you, Bill. I'll add that one...

[the remark this answers is not in the document]
Comment (lizz7000) [reply content/outcomes/application-development-rhythm/foundations.md:152]: Hi Bill,

I'm still looking at the patterns topic. When you write, "Think of patterns as one of the 
high level *whats* of building software.", it's a little unclear to me what a high level "what" might be. Can you please provide an example in your description? 

Also, can you please provide a "why" to the sentence, *Patterns* are also long-lived, similar to *practices* in the same way that you did when describing the "how" in the Practices section?

**Comment by lizz7000** [comment]
Hi Bill,

Please see the following sentence under the Practices section.

"The lifetime of principles is tied to the relevance of the context in which they
live."

I understand what you're saying, but I have to stop and think about it. If you had to describe this as simply as possible, how would you write it?

**Comment by joemoore** [comment]
Is this pull request still relevant? If not we can close it.

**Comment by lizz7000** [comment]
Hi,

It can be closed.

Thanks,

Lizz.
________________________________

**Comment by joemoore** [comment]
closed by request

**Pull request closed without merging**